index.html

```diff
@@ -1873,74 +1880,97 @@ <h2>The <dfn>record type</dfn> string</h2>
     section is as follows:
   </p>
-  <table class="simple">
+  <table class="simple" data-link-for="NDEFRecordInit">
     <tr>
-      <th>NDEFRecordInit recordType</th>
-      <th>NDEFRecordInit mediaType</th>
-      <th>NDEFRecordInit data</th>
-      <th>NDEF record type</th>
+      <th>[=recordType=]</th>
+      <th>[=mediaType=]</th>
+      <th>[=data=]</th>
+      <th><a href="#ndef-record-types">record type</a></th>
+      <th nowrap>[=TNF field=]</th>
+      <th>[=TYPE field=]</th>
     </tr>
     <tr>
       <td><dfn>"`empty`"</dfn></td>
       <td><i>unused</i></td>
       <td><i>unused</i></td>
-      <td><a>Empty record</a> (TNF=0)</td>
+      <td><a>Empty record</a></td>
+      <td>0</td>
+      <td><i>unused</i></td>
     </tr>
     <tr>
       <td><dfn>"`text`"</dfn></td>
       <td><i>unused</i></td>
       <td>{{DOMString}}</td>
-      <td><a>Well-known type</a> (TNF=1) record with TYPE="`T`"</td>
+      <td><a>Well-known type</a></td>
+      <td>1</td>
+      <td>"`T`"</td>
     </tr>
     <tr>
       <td><dfn>"`url`"</dfn></td>
       <td><i>unused</i></td>
       <td>{{DOMString}}</td>
-      <td><a>Well-known type</a> (TNF=1) record with TYPE="`U`"</td>
+      <td><a>Well-known type</a></td>
+      <td>1</td>
+      <td>"`U`"</td>
     </tr>
     <tr>
       <td nowrap><dfn>"`smart-poster`"</dfn></td>
       <td><i>unused</i></td>
       <td>{{NDEFMessage}}</td>
-      <td><a>Well-known type</a> (TNF=1) record with TYPE="`Sp`"</td>
+      <td><a>Well-known type</a></td>
+      <td>1</td>
+      <td>"`Sp`"</td>
     </tr>
     <tr>
       <td><dfn>"`media`"</dfn></td>
       <td>[= MIME type =]</a></td>
       <td>{{BufferSource}}</td>
-      <td><a>MIME type record</a> (TNF=2) with TYPE= [= MIME type =]</td>
+      <td><a>MIME type record</a></td>
+      <td>2</td>
+      <td>[= MIME type =]</td>
     </tr>
     <tr>
       <td nowrap><dfn>"`absolute-url`"</dfn></td>
       <td><i>unused</i></td>
       <td>{{DOMString}} <i>url</i></td>
-      <td><a>Absolute-URL record</a> (TNF=3) with TYPE=<i>url</i></td>
+      <td><a>Absolute-URL record</a></td>
+      <td>3</td>
+      <td>[=Absolute-URL=]</td>
     </tr>
     <tr>
-      <td><i>[= external type =]</i></td>
+      <td><i>[=external type name=]</i></td>
       <td><i>unused</i></td>
       <td>
         {{BufferSource}} or<br>
         {{NDEFMessage}}
       </td>
-      <td><a>External type record</a> (TNF=4) with TYPE= [= external type =]</td>
+      <td><a>External type record</a></td>
+      <td>4</td>
+      <td>[=external type name=]</td>
     </tr>
     <tr>
-      <td><i>[= local type =]</i></td>
+      <td><i>[=local type name=]</i></td>
       <td><i>unused</i></td>
       <td>
         {{BufferSource}}
       </td>
-      <td><a>External type record</a> (TNF=4) with TYPE= [= local type =] inside
-        the <a>NDEFMessage</a> payload of another record</td>
+      <td>[=Local type=] record*</td>
```

Comment:
We should add a section called "Local type record" - it is missing

Reply:
Since it seems to be ordered by TNF, shall we move "local type" row under "smart-poster" row?